pkg/webhook/policy.go

```diff
@@ -56,7 +56,11 @@ var (
 
 // AddPolicyWebhook registers the policy webhook server with the manager
 func AddPolicyWebhook(mgr manager.Manager, opa *opa.Client) error {
-	wh := &admission.Webhook{Handler: &validationHandler{opa: opa, client: mgr.GetClient()}}
+	reporter, err := newStatsReporter()
+	if err != nil {
+		panic(err)
```

Comment:
can we return the err? would that also call panic?

Reply:
Good catch.

Not panic exactly, but it would cause the main process to quit, which is the goal.

Updated.